### PR TITLE
Send translations

### PR DESCRIPTION
**Pull request by davidmason**
Send translations to Zanata if any are found for each source document.

### CODE REVIEW

File: tmgmt_zanata.connector.inc [hunk 1]
```diff
@@ -362,9 +367,230 @@ class TmgmtZanataConnector {
...
+                '@url' => $url));
+            return FALSE;
+        }
+      }
+      else {
+        // no existing translations found. Probably don't need a log message.
+      }
+
+    }
+    else {
```
Comment (huangp) [inline tmgmt_zanata.connector.inc]: I would suggest invert this condition and let it test at the top of the function and return immediately if goes into the block. e.g.
```
if (!$is_node) {
    ...
   return FALSE;
}
```
Then you can move the original if block outside (no need to wrap it in a condition check block). Will make it easier to read.

File: tmgmt_zanata.connector.inc [hunk 2]
```diff
@@ -362,9 +367,230 @@ class TmgmtZanataConnector {
...
+    );
+  }
+
+  private function getExistingTransData(TMGMTJobItem $job_item) {
+    $id = $job_item->item_id;
+    $node = node_load($id);
+
+    $translations = translation_node_get_translations($node->tnid);
+
+    if ($translations) {
```
Comment (huangp) [inline tmgmt_zanata.connector.inc]: Same practice should apply here. See http://www.codeproject.com/Articles/626403/How-and-Why-to-Avoid-Excessive-Nesting (it use continue but here we would use return)

**Comment by huangp** [comment]
Finished review. Nothing major except the two suggested refactor.

File: tmgmt_zanata.connector.inc (hunk 1 above)
Comment (davidmason) [inline tmgmt_zanata.connector.inc]: Good suggestion, I will do that.
- [ ] change `else` portion to a check that returns if failed.

File: tmgmt_zanata.connector.inc (hunk 2 above)
Comment (davidmason) [inline tmgmt_zanata.connector.inc]: - [ ] Change translation check else portion to a pre-check that returns if failed.

**Comment by huangp** [comment]
:+1:

**Pull request merged**